build-dakkar.sh

```diff
@@ -145,6 +146,13 @@ function get_rom_type() {
                 treble_generate="slim"
                 extra_make_options="WITHOUT_CHECK_API=true"
                 ;;
+	    tipsy)
+                mainrepo="https://github.com/TipsyOs/platform_manifest.git "
```

Comment:
You should delete the trailing space

```suggestion
                mainrepo="https://github.com/TipsyOs/platform_manifest.git"
```